### ISSUE TITLE
Bogus lispusers files

### ISSUE THREAD

**Issue by rmkaplan** [Interlisp/medley#667]
While scanning for string-arguments to TEDIT and OPENTEXTSTREAM, I encountered a number of lispusers files that don't load.  Some of them (INDEX, NGROUP) I think should be moved to obsolete/ (they are replaced by TMAX-INDEX and TMAX-NGROUP).

The problem with others is that they have FILES commands that specify particular hosts and directories (e.g. {ERIS}) that no longer exist.  For most of those I presume the intent is (VALUEOF LISPUSERSDIRECTORIES)).

A few of them (MIME, MTP...) are subpackages of Lafite, and they are explicitly trying to get Lafite from a place where Lafite no longer resides.  Lafite is now in library/lafite/.  Maybe they should be moved from lispusers/ to library/lafite/?  Or maybe we need a variable LAFITEDIRECTORIES?

(Finally, does the file DICTCLIENT exist anywhere in the archives?  This was part of the Dictionary Server--some of its files remain in Lispusers but they depend on this.  Maybe all the dictserver files should also go to obsolete.  The dictclient was probably something that was running on a separate server that held the old dictionary data files.)

**Comment by nbriggs** [comment]
LISPNERD loads `(FROM {NFS}<TILDE>DICTSERVER>LISP>) DICTCLIENT)` so I would guess we don't have it.  Was it an implementation of the [RFC 2229 DICT server](https://datatracker.ietf.org/doc/html/rfc2229) protocol?

**Comment by Anzus** [comment]
Probably some sort of dictionary server, but that RFC came out in 1997 :-).

**Comment by nbriggs** [comment]
... or the webster protocol, which predated the DICT protocol.   I remember the dict server running at PARC, but never investigated the details.  I think it had the Webster's dictionary as the basis, and it appears it also had the IRM for the "lispnerd" lookups.

**Comment by rmkaplan** [comment]
No, it was something that we did internally, John Maxwell, Martin and me, and maybe WIllie Crowther. We got early tapes of the American Heritage Dictionary  (or maybe it was Merriam Webster, now I don’t remember).  We put it up as a service, and maybe used it also for remote spell checking in those days.  (It turned out that the latter half of the letter g was written past the end of tape mark, we went back to them and they wouldn’t give it under our original arrangement—by that time they had realized that there was money to be made on line.)

We also set it up as what we called the “wordnerd”, the capability of looking up a word up by its definition (kind of radical in those days).  “plastic on the end of a shoe lace” —> aglet.  “mechanical model of the solar system” -> orrery.  One of the first inverted indexes—we filed a patent on indexed searching, but the patent examiner said it was no different from what you find at the back of a book.

Sometime in the 80’s, well before any protocols (or maybe we didn’t know about them).

So I think this is clearly defunct, I don’t think we can resurrect it, and there would be no reason to.  Is there any reason not to move it to obsolete, for historians?  It would be nice to recover the DICTCLIENT code, to complete the artifact, but there is no value in tripping over it in the current lispusers.

**Comment by rmkaplan** [comment]
Actually, I think Willie was interested because he needed access to words of various sorts for his original Adventure game.

**Comment by nbriggs** [comment]
If someone still at PARC can see the old file server backups they might be able to recover things from either /tilde/dictserver or /project/dictserver -- (or it might still be spinning somewhere in the building).

**Comment by rmkaplan** [comment]
I’ll ask John

**Comment by masinter** [comment]
I'm comfortable with having lispusers files that were either released by Venue, owned by Venue or were contributed anew. 
Let's remove clutter.

**Comment by pamoroso** [comment]
INDEX, NGROUP, and MTP were eventually moved to obsolete.

Are MIME and the dictionary server the remaining files of this issue that should be moved to obsolete?

**Comment by pamoroso** [comment]
MIME is now in library/lafite, so it's probably no longer relevant to this lispusers issue.
